File: 23-02-2026/string_mirror.py
def mirror(s):
    return s + s[::-1]

print(mirror("The quick brown fox...")); # "The quick brown fox... ...xof nworb kciuq ehT"
print(mirror("helloworld")); # "helloworlddlrowolleh"
print(mirror("RaceCar")); # "RaceCarraCecaR"
print(mirror("freeCodeCamp")); # "freeCodeCamppmaCedoCeerf"

File: 23-02-2026/string_mirror.js
function mirror(str) {
  return `${str}${str.split("").reverse().join("")}`;
}

console.log(mirror("The quick brown fox...")); // "The quick brown fox... ...xof nworb kciuq ehT"
console.log(mirror("helloworld")); // "helloworlddlrowolleh"
console.log(mirror("RaceCar")); // "RaceCarraCecaR"
console.log(mirror("freeCodeCamp")); // "freeCodeCamppmaCedoCeerf"
